src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationParameters.cs

```diff
@@ -60,13 +61,20 @@ public sealed class AuthenticationParameters
         /// </summary>
         public string Resource { get; set; }
 
+        private static IHttpManager HttpManager { get; set; }
```

Comment:
You don't need a settter if you only set it from the ctor. In fact, it would be better if this was a readonly member.